selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/RenameWorkspaceTest.java

```diff
@@ -51,7 +52,7 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    workspaceServiceClient.delete(ws.getName(), user.getName());
+    workspaceServiceClient.delete(MAX_WORKSPACE_NAME_SIZE, user.getName());
```

Comment:
Please, process removal by the all possible workspace names including **MIN_WORKSPACE_NAME**, wotkspace names with size 1 and 101.